index.html

```diff
@@ -0,0 +1,221 @@
+            <div class="about-text">
+                <h2>About me</h2>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <button class="about-button" type="button">
+                    Get My Resume
+                </button>
+                <div class="about-background"></div>
+            </div>
+            <hr>
+            <div class="about-language">
+                <ul>
+                    <li>Languages
+                        <div class="sublist">
+                            <span class="sublist-title">Javascript</span>
+                            <span class="sublist-title">Ruby</span>
+                            <span class="sublist-title">Html</span>
+                            <span class="sublist-title">Css</span>
+                        </div>
+                    </li>
+                    <li>Frameworks
+                        <div class="sublist">
+                            <span class="sublist-title">Bootstrap</span>
+                            <span class="sublist-title">Ruby on Rails</span>
+                            <span class="sublist-title">RSPec</span>
+                            <span class="sublist-title">CapyBara</span>
+                            <span class="sublist-title">Selenium</span>
+                        </div>
+                    </li>
+                    <li>Skills
+                        <div class="sublist">
+                            <span class="sublist-title">Codekit</span>
+                            <span class="sublist-title">Github</span>
+                            <span class="sublist-title">Codepen</span>
+                            <span class="sublist-title">Gitlab</span>
+                            <span class="sublist-title">Terminal</span>
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="section-contact">
+            <div class="contact-text">
+                <p>
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </p>
+
+            </div>
+            <div class="contact-form">
+                    <form action="https://formspree.io/f/moqybakj" method="POST">
+                        <input type="text" placeholder="Full Name" required maxlength="30" >
+                        <input type="email" placeholder="Email" maxlength="30">
+                        <textarea name="message" id="message" cols="38" rows="7" maxlength="500" placeholder="Enter text here"></textarea>
+                        <input type="submit" value="Get in touch" class="button">
+                      </form>
+            </div>
```

Comment:
Form width should be a bit large than the text in the contact section. Moreover please use the same font color for text which is already defined on the design page. KIndly see the screenshot below.
![form](https://user-images.githubusercontent.com/35504149/135471868-05598fbd-c1f3-4de0-a765-b52325410ab6.png)